Dockerfile

```diff
@@ -48,69 +47,72 @@ RUN apt-get -y install \
     latex-cjk-chinese-arphic-gbsn00lp latex-cjk-chinese-arphic-gkai00mp \
     fonts-symbola
 
-# Install 'rsvg-convert' for SVG -> PDF conversion
+# plantuml: is to support sphinxcontrib-plantuml
+# https://pypi.org/project/sphinxcontrib-plantuml/
+#
+# imagemagick: is to support sphinx.ext.imgconverter
+# http://www.sphinx-doc.org/en/master/usage/extensions/imgconverter.html
+#
+# rsvg-convert: is for SVG -> PDF conversion
 # using Sphinx extension sphinxcontrib.rsvgconverter, see
 # https://github.com/missinglinkelectronics/sphinxcontrib-svg2pdfconverter
-RUN apt-get -y install librsvg2-bin
-
-# Install plantuml
-RUN apt-get -y install plantuml
+#
+# swig: is required for different purposes
+# https://github.com/rtfd/readthedocs-docker-images/issues/15
+RUN apt-get -y install plantuml imagemagick librsvg2-bin swig
 
 # Install Python tools/libs
 RUN apt-get -y install python-pip && pip install -U virtualenv auxlib
 
+# Install jsdoc and typedoc
+RUN apt-get -y install nodejs npm && npm install --global jsdoc typedoc
+
 # UID and GID from readthedocs/user
 RUN groupadd --gid 205 docs
 RUN useradd -m --uid 1005 --gid 205 docs
 
-# Install jsdoc and typedoc
-RUN apt-get -y install nodejs npm && npm install --global jsdoc typedoc
-
 USER docs
 WORKDIR /home/docs
 
 # Install Conda
-RUN curl -O https://repo.continuum.io/miniconda/Miniconda2-${CONDA_VERSION}-Linux-x86_64.sh
-RUN bash Miniconda2-${CONDA_VERSION}-Linux-x86_64.sh -b -p /home/docs/.conda/
+RUN curl -O https://repo.continuum.io/miniconda/Miniconda3-${CONDA_VERSION}-Linux-x86_64.sh
+RUN bash Miniconda3-${CONDA_VERSION}-Linux-x86_64.sh -b -p /home/docs/.conda/
 ENV PATH $PATH:/home/docs/.conda/bin
+RUN rm -f Miniconda3-${CONDA_VERSION}-Linux-x86_64.sh
```

Comment:
Is miniconda3 a python3 binary? does this require us to make sure the project is using python3 before executing? If so, do we have a similar .org change that prevents breaking existing python2 conda projects?

Reply:
Perhaps use both miniconda2 and miniconda3? or are they not compatible?

Reply:
I will research a little more about this. I'm not sure what should be the flow. I'm creating a ticket for this since it will be something new and unrelated with this PR.

Reply:
We have a problem either way it seems. For now, this image will be correct, but when we move to default python3, new projects will have to specify python2 to use conda.

Reply:
I don't think so. The `project.python_interpreter` is not used at all when building with `conda`. It's only used to create the virtual environment.